node/chainSimulator/components/processComponents.go

```diff
@@ -96,7 +96,7 @@ type processComponentsHolder struct {
 }
 
 // CreateProcessComponentsHolder will create the process components holder
-func CreateProcessComponentsHolder(args ArgsProcessComponentsHolder) (factory.ProcessComponentsHolder, error) {
+func CreateProcessComponentsHolder(args ArgsProcessComponentsHolder) (factory.ProcessComponentsHandler, error) {
```

Comment:
Please rename to `CreateProcessComponents` and remove the inner closeHandler declaration & usage

Reply:
renamed